### ISSUE TITLE
`inv manage createsuperuser` cannot run due to not being in TTY mode

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#1530]
It's probably a good idea to create a shorthand task for the `createsuperuser` command, if possible (it looks like `inv` might make TTY input impossible?)

**Comment by snahil** [comment]
Yes, this will be a great idea as you never gets a terminal with TTY terminal input
